### PR TITLE
Add kubectl debug video.

### PR DESCRIPTION
**Pull request by amouat**
## Type of change
Landing page for kubectl debug video

### CODE REVIEW

File: content/chainguard/chainguard-images/videos/kubectl_cdebug.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
+---
+title: "Debugging Distroless Images with Kubectl Debug and CDebug"
+linktitle: "Debugging with Kubectl and CDebug"
+lead: ""
+description: "This video explains how to use the Kubectl and CDebug tools to investigate
```
Comment (smythp) [inline content/chainguard/chainguard-images/videos/kubectl_cdebug.md:5]: I feel like I mostly see cdebug all lower case. Kubectl seems to come up both ways.

File: content/chainguard/chainguard-images/videos/kubectl_cdebug.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+What you really want to do in this case is run your containers not as the root user but as different user -- as the Chainguard nginx image runs.
+
+So the Chainguard nginx defines a non-root user that's running that and that's you know security best practice.
+
+And what we want is we want our debug container to also run as the same user.
+
+So how can we do that?
+
+So in this case we've used alpine image but we can change to use different image.
```
Comment (smythp) [inline content/chainguard/chainguard-images/videos/kubectl_cdebug.md:129]: a different

File: content/chainguard/chainguard-images/videos/kubectl_cdebug.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+Now what happens if I access the file system this time?
+
+Looks like I can't see it but if I open the file, no I put a slash in my mistake.
+
+So I can look at the file but in this case I can't edit it.
+
+But still a pretty good result all I had to do is pass the `--user` flag.
+
+Okay so that was both cdebug and `kubectl debug` which are both great utilities for debugging distro containers.
```
Comment (smythp) [inline content/chainguard/chainguard-images/videos/kubectl_cdebug.md:197]: distro containers > distroless containers

**Review by smythp — APPROVED** [review]
This makes me feel like I should have done a video on cdebug after I got through playing around with it. Good video.

Mostly I feel like it's not worth going through transcripts with a fine-tooth comb but if something is confusing for understanding I think it's worth changing. In this case really just the "distro containers" at the end.

File: content/chainguard/chainguard-images/videos/kubectl_cdebug.md (hunk 1 above)
Comment (amouat) [reply content/chainguard/chainguard-images/videos/kubectl_cdebug.md:5]: Yeah, but it's CDebug in the title of the video which I can't change :/

Comment (amouat) [reply content/chainguard/chainguard-images/videos/kubectl_cdebug.md:5]: I've kept it as CDebug in titles and changed in body text. Not sure if that's more confusing 😅

**Pull request merged**